README.md

```diff
@@ -1,20 +1,12 @@
 # Exomiser Runner for PhEval
 
-This is the Exomiser plugin for PhEval. Highly experimental. Do not use.
+This is the Exomiser plugin for PhEval. Documentation on how to set up and run the PhEval pipeline with the Exomiser runner is detailed [here](https://monarch-initiative.github.io/pheval/exomiser_pipeline/).
```

Comment:
Explain briefly how this differs from the PheEval pipeline. i.e. that there needs to be a prepare step before hand to prepare the corpora for analysis.